profile.html

```diff
@@ -15,7 +15,13 @@
   <body class="wrapper">
     <header class="header">Profile</header>
     <main class="main-main">
-      <div class="profile"><i class="im im-user-male kopf">Max Meyer</i></div>
+      <div class="profile img-profile">
```

Comment:
I am not a fan of `img-profile`. Sounds like a profile of an image (which does not make sense to me).